app/controllers/api/braintree_controller.rb

```diff
@@ -1,31 +1,80 @@
 class Api::BraintreeController < ApplicationController
+  skip_before_action :verify_authenticity_token
```

Comment:
Yes, but I like to have it in for now since I post through the command line
all the time.

On Wed, Nov 25, 2015 at 10:16 AM, NealJMD <notifications@github.com> wrote:

> In app/controllers/api/braintree_controller.rb
> <https://github.com/SumOfUs/Champaign/pull/242#discussion_r45900961>:
>
> > @@ -1,31 +1,80 @@
> >  class Api::BraintreeController < ApplicationController
> > +  skip_before_action :verify_authenticity_token
>
> this still needs to go too right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SumOfUs/Champaign/pull/242/files#r45900961>.
>